dnf/cli/commands/install.py

```diff
@@ -61,9 +61,15 @@ def run(self):
         strict = self.base.conf.strict
 
         # Install files.
-        # try to install packages with higher version first
-        for pkg in sorted(self.base.add_remote_rpms(self.opts.filenames, strict=strict), reverse=True):
-            self.base.package_install(pkg, strict=strict)
+        err_pkgs = []
+        for pkg in self.base.add_remote_rpms(self.opts.filenames, strict=strict):
+            try:
+                self.base.package_install(pkg, strict=strict)
+            except dnf.exceptions.MarkingError:
+                msg = _('No match for argument: %s%s%s')
```

Comment:
Another point is _('No match for argument: %s%s%s') and _('No match for argument: %s') where both cases are in the code. I can refactor all into something like 
msg = ' '.join(_('No match for argument:'), package)
msg = ' '.join(_('No match for argument:'), ''.join(self.base.output.term.MODE['bold'], package, self.base.output.term.MODE['normal']))